scripts/ci/config.yml

```diff
@@ -96,7 +97,8 @@ mlflow:
   override_name_or_path: mlflow
   dependencies:
     - "mlflow"
-    - "scikit-learn"
+  external_scripts: |
+    python -m pip install -U scikit-learn
```

Comment:
what's the reason for moving this change?

Reply:
I think the version for scikit-learn from mlflow is older and it doesn't have some of the internal library